internal/code-generator/generator/generator.go

```diff
@@ -1,3 +1,5 @@
+// Package generator provides facilities to generate Golang code from data in
```

Comment:
```suggestion
// Package generator provides facilities to generate Go code from data in
```

Reply:
Could you please be more specific in the comment about which data you mean?

Reply:
Would something like `Package generator provides facilities to generate Go code in data package from ...` work?

Otherwise I'm afraid it would be require to explain whole code-generation approach here, which I'm happy to do but outside of this PR.